firestore-bigquery-export/scripts/import/src/index.ts

```diff
@@ -59,10 +60,44 @@ const validateInput = (
   return true;
 };
 
+const validateBatchSize = (value: string) => {
+  return parseInt(value, 10) > 0;
+};
+
+program
+  .name("fs-bq-import-collection")
+  .option(
+    "--non-interactive",
+    "Parse all input from command line flags instead of prompting the caller.",
+    false
+  )
+  .option(
+    "-P, --project <project>",
+    "Firebase Project ID for project containing the Cloud Firestore database."
+  )
+  .option(
+    "-s, --source-collection-path <source-collection-path>",
+    "The path of the the Cloud Firestore Collection to import from. (This may, or may not, be the same Collection for which you plan to mirror changes.)"
+  )
+  .option(
+    "-d, --dataset <dataset>",
+    "The ID of the BigQuery dataset to import to. (A dataset will be created if it doesn't already exist.)"
+  )
+  .option(
+    "-t, --table-name-prefix <table-name-prefix>",
+    "The identifying prefix of the BigQuery table to import to. (A table will be created if one doesn't already exist.)"
+  )
+  .option(
+    "-b, --batch-size [batch-size]",
+    "Number of documents to stream into BigQuery at once.",
+    (value) => parseInt(value, 10),
+    300
+  );
+
 const questions = [
   {
     message: "What is your Firebase project ID?",
-    name: "projectId",
+    name: "project",
```

Comment:
Is there any reason these have been changed to remove the `Id` postfix?

Reply:
Yes.
I wanted the CLI to match that of `gen-schema-view`, which doesn't use the `Id` postfix.
Once I had the CLI using the non postfixed versions I wanted the prompts to match the CLI parameters.


`gen-schema-view`  CLI options are:
```
Usage: gen-schema-views [options]

Options:
  --non-interactive                            Parse all input from command line flags instead of prompting the caller. (default: false)
  -P, --project <project>                      Firebase Project ID for project containing Cloud Firestore database.
  -d, --dataset <dataset>                      The ID of the BigQuery dataset containing a raw Cloud Firestore document changelog.
  -t, --table-name-prefix <table-name-prefix>  A common prefix for the names of all views generated by this script.
  -f, --schema-files <schema-files>            A collection of files from which to read schemas. (default: [])
  -h, --help                                   display help for command
```